src/ROUTES.js

```diff
@@ -68,7 +68,9 @@ export default {
     REPORT,
     REPORT_WITH_ID: 'r/:reportID?',
     EDIT_REQUEST: 'r/:threadReportID/edit/:field',
+    EDIT_CURRENCY_REQUEST: 'r/:threadReportID/edit/currency',
```

Comment:
Can you move this line after `getEditRequestRoute`? It's easier to see the pair together like the examples below this one